react-common/styles/onboarding/TeachingBubble.less

```diff
@@ -98,6 +98,11 @@
         }
     }
 
+    .common-button:focus-visible {
+        outline: 2px solid var(--pxt-tertiary-foreground);
```

Comment:
Did you experiment with var(--pxt-focus-border) for this color? Ideally, we'd use that in most cases where we're showing focus like this. If it means we have to change the color --pxt-focus-border is set to, that's definitely an option.